ranger/data/scope.sh

```diff
@@ -160,7 +160,26 @@ handle_image() {
         #     exit 1;;
 
         ## PDF
-        # application/pdf)
+        application/pdf)
+            if [[ "$(uname)" = "Darwin" ]]; then
+                ## Use mac built-in quicklook to preview pdf
+                ## rename extension from .png to .jpg provided by IMAGE_CACHE_PATH
+                ## image viewer can determine the actual format
+                qlmanage -t -s "${DEFAULT_SIZE%x*}" \
+                         -o "${IMAGE_CACHE_PATH%/*}" \
+                         ${FILE_PATH} \
+                         && mv "${IMAGE_CACHE_PATH%/*}/${FILE_PATH##*/}.png" "${IMAGE_CACHE_PATH}" \
```

Comment:
We could use a temporary file like elsewhere in `scope.sh`. That way we don't put anything in the cache that isn't used as a cached preview.

Reply:
I could change this. But why do you want to put into tmp first? I don't get it.

Reply:
It's just cleaner. The cache directory should only contain finished previews ideally.